src/domain/community/virtualContributor/vcProfilePage/VCProfileContentView.tsx

```
@@ -154,3 +202,34 @@ export const VCProfileContentView = ({
+  );
+}
+
+function SectionWrapper({ children }: PropsWithChildren<{}>) {
+  return (
+    <Gutters disablePadding sx={{ flexDirection: 'row', flexWrap: 'wrap' }}>
+      {children}
+    </Gutters>
+  );
+}
+
+function SectionItem({ children }: PropsWithChildren<{}>) {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)